internal/gps/pkgtree/pkgtree.go

```diff
@@ -203,6 +203,12 @@ func fillPackage(p *build.Package) error {
 		if filepath.Base(file)[0] == '_' {
 			continue
 		}
+
```

Comment:
Also, that still leaves us open to the case where somebody names a directory "foo.go" inside their project. Perhaps we should handle that case separately, or at least give them a more helpful error message?